api/core/domain/models/model_datas_mapping.py

```diff
@@ -1025,6 +1025,42 @@ def _build_model_datas():
             provider_name=DisplayedProvider.FIREWORKS.value,
             supports_tool_calling=False,
         ),
+        Model.QWEN3_235B_A22B: ModelData(
+            display_name="Qwen3 235B (22B active)",
+            supports_json_mode=True,
+            supports_input_image=False,  # Double check
+            supports_input_pdf=False,
+            supports_input_audio=False,
+            supports_structured_output=True,
+            max_tokens_data=MaxTokensData(
+                max_tokens=32_768,
+                source="https://huggingface.co/Qwen/Qwen3-235B-A22B",
+            ),
+            provider_for_pricing=Provider.FIREWORKS,
+            icon_url="https://workflowai.blob.core.windows.net/workflowai-public/qwen.svg",
+            release_date=date(2025, 4, 29),
+            quality_index=789,  # GPQA = 70 MMLU = 87.8 https://qwenlm.github.io/blog/qwen3/, https://x.com/ArtificialAnlys/status/1917246369510879280
+            provider_name=DisplayedProvider.FIREWORKS.value,
+            supports_tool_calling=False,
+        ),
+        Model.QWEN3_30B_A3B: ModelData(
+            display_name="Qwen3 30B (3B active)",
+            supports_json_mode=True,
+            supports_input_image=False,
+            supports_input_pdf=False,
+            supports_input_audio=False,
+            supports_structured_output=True,
+            max_tokens_data=MaxTokensData(
+                max_tokens=32_768,
```

Comment:
```suggestion
                max_tokens=128_000,
```
?
<img width="433" alt="Screenshot 2025-04-30 at 11 05 52 AM" src="https://github.com/user-attachments/assets/c62d41ee-faf2-49ba-a33f-74e63c597547" />